_data/nav.yml

```diff
@@ -1,10 +1,34 @@
 - title: "Home"
   href: "/"
 
-- title: "About"
+
+- title: Attend
+  subcategories:
+    - subtitle: Schedule
+      subhref: /schedule
+    - subtitle: Tickets
+      subhref: /tickets
+
+- title: "Sponsors & Friends"
+  subcategories:
+    - subtitle: "Sponsors"
+      subhref: "/sponsors"
+    - subtitle: "Friends"
+      subhref: "/friends"
+
+- title: "Volunteers & Presenters"
+  subcategories:
+    - subtitle: "Volunteers"
+      subhref: "/volunteer"
+    - subtitle: "Presenters"
+      subhref: "/cfp"
+    - subtitle: "Organizers"
```

Comment:
Should we even include this if we're getting rid of this for now?

Reply:
I'm open to either keeping or removing.  I'd like something on the nav for people to watch, and in future apply when we open up for more to join us, without them having to remember /jobs